src/game.zig

```diff
@@ -156,6 +158,146 @@ pub const Game = struct {
+    var game = try Game.init(10, std.testing.allocator);
+    defer game.deinit();
+
+    _ = try game.place(.{ 1, 1 }, .Player1);
+    _ = try game.place(.{ 2, 2 }, .Player1);
+    _ = try game.place(.{ 3, 3 }, .Player1);
+    _ = try game.place(.{ 4, 4 }, .Player1);
+    _ = try game.place(.{ 5, 5 }, .Player1);
+    try std.testing.expect(game.is_move_winning(.{ 5, 5 }));
+    try std.testing.expect(game.is_move_winning(.{ 1, 1 }));
+}
+
+test "is_move_winning detects Anti-diagonal win" {
+    var game = try Game.init(10, std.testing.allocator);
+    defer game.deinit();
+
+    _ = try game.place(.{ 8, 1 }, .Player2);
+    _ = try game.place(.{ 7, 2 }, .Player2);
+    _ = try game.place(.{ 6, 3 }, .Player2);
+    _ = try game.place(.{ 5, 4 }, .Player2);
+    _ = try game.place(.{ 4, 5 }, .Player2);
+    try std.testing.expect(game.is_move_winning(.{ 7, 2 }));
+}
+
+test "is_move_winning detects No win (incomplete line)" {
+    var game = try Game.init(10, std.testing.allocator);
+    defer game.deinit();
+
+    _ = try game.place(.{ 2, 5 }, .Player1);
+    _ = try game.place(.{ 3, 5 }, .Player1);
+    _ = try game.place(.{ 4, 5 }, .Player1);
+    _ = try game.place(.{ 5, 5 }, .Player1);
+    try std.testing.expect(!game.is_move_winning(.{ 5, 5 }));
+}
+
+test "is_move_winning does not crash on near border check" {
+    var game = try Game.init(10, std.testing.allocator);
+    defer game.deinit();
+
+    _ = try game.place(.{ 0, 1 }, .Player1);
+    _ = try game.place(.{ 1, 2 }, .Player1);
+    _ = try game.place(.{ 2, 3 }, .Player1);
+    _ = try game.place(.{ 3, 4 }, .Player1);
+    _ = try game.place(.{ 4, 5 }, .Player1);
+    try std.testing.expect(game.is_move_winning(.{ 1, 2 }));
+}
+
+test "places a valid move that doesn't win" {
+    var game = try Game.init(10, std.testing.allocator);
+    defer game.deinit();
+
+    const is_win = try game.place(.{ 5, 5 }, .Player1);
+    try std.testing.expectEqual(false, is_win);
+    try std.testing.expectEqual(CellState.Player1, game.at(.{ 5, 5 }));
+}
+
+test "places on a cell that is already taken" {
+    var game = try Game.init(10, std.testing.allocator);
+    defer game.deinit();
+
+    _ = try game.place(.{ 5, 5 }, .Player1);
+    try std.testing.expectError(Error.AlreadyTaken, game.place(.{ 5, 5 }, .Player2));
+}
+
+test "places on a cell out of the game bounds" {
+    var game = try Game.init(10, std.testing.allocator);
+    defer game.deinit();
+
+    try std.testing.expectError(Error.OutOfBound, game.place(.{ 11, 11 }, .Player2));
+}
+
+test "places a move that leads to a win" {
+    var game = try Game.init(10, std.testing.allocator);
+    defer game.deinit();
+
+    _ = try game.place(.{ 2, 5 }, .Player1);
+    _ = try game.place(.{ 3, 5 }, .Player1);
+    _ = try game.place(.{ 4, 5 }, .Player1);
+    _ = try game.place(.{ 5, 5 }, .Player1);
+    const is_win = try game.place(.{ 6, 5 }, .Player1);
+    try std.testing.expectEqual(true, is_win);
```

Comment:
Inline those two lines together, there is no need to have them split

```suggestion
    try std.testing.expect(try game.place(.{ 6, 5 }, .Player1));
```